README.md

```diff
@@ -61,7 +62,7 @@ defmodule Membrane.ReleaseTest.Pipeline do
           sample_rate: 48_000
         }
       },
-      serializer: Membrane.Opus.Serializer,
+      parser: Membrane.Opus.Parser,
```

Comment:
```suggestion
      parser: %Membrane.Opus.Parser{delimitation: :delimit},
```

Actually, Opus has two versions of packets: non-delimited (default) and delimited. The delimited version allows determining where each packet ends and another one begins, making it possible to just write the stream to a file that is playable (though media players usually don't support that and require some container, like Ogg).